builder/vmware/common/ssh_config.go

```diff
@@ -6,21 +6,22 @@
 package common
 
 import (
+	"log"
+
 	"github.com/hashicorp/packer-plugin-sdk/communicator"
 	"github.com/hashicorp/packer-plugin-sdk/template/interpolate"
 )
 
 type SSHConfig struct {
 	Comm communicator.Config `mapstructure:",squash"`
 
-	// These are deprecated, but we keep them around for BC
-	// TODO(@mitchellh): remove
+	// TODO: Deprecated. Remove in next major release
 	SSHSkipRequestPty bool `mapstructure:"ssh_skip_request_pty"`
 }
 
 func (c *SSHConfig) Prepare(ctx *interpolate.Context) []error {
 	if c.SSHSkipRequestPty {
-		c.Comm.SSHPty = false
```

Comment:
Is there a reason why we remove this option? If this is a no-op, no problem with removing this statement, otherwise I would still keep that assignation alive for now until we remove it completely, to avoid accidentally breaking existing templates

Reply:
Sorry, I mean to to add that back in the prior fixup!

Resolved in 54ccaa3 .